OpenRA.Game/Map/Map.cs

```diff
@@ -151,8 +151,7 @@ public class Map
 		};
 
 		public const int MaxTilesInCircleRange = 50;
-		public readonly TileShape TileShape;
-		public readonly byte MaximumTerrainHeight;
+		public readonly MapGridData MapGrid;
```

Comment:
We're already in `Map`, so you can simplify this down to `Grid` to remove duplication everywhere that uses it.